build/configs/rtl8730e/rtl8730e_download.sh

```diff
@@ -58,12 +59,35 @@ function pre_download()
 	if test -f "${BIN_PATH}/${BOOTPARAM}.bin"; then
 		cp -p ${BIN_PATH}/${BOOTPARAM}.bin ${IMG_TOOL_PATH}/${BOOTPARAM}.bin
 	fi
+	if test -f "${BIN_PATH}/${EXTERNAL}.bin"; then
+		cp -p ${BIN_PATH}/${EXTERNAL}.bin ${IMG_TOOL_PATH}/${EXTERNAL}.bin
+	fi
 
 	echo ""
 	echo "=========================="
-	echo "Checking flash size"
+	echo "Checking flash size(PASS)"
```

Comment:
What does `(PASS)` mean?

Reply:
We dont need to check for 16MB verification. It is always 16 MB